op-supervisor/supervisor/backend/db/db.go

```diff
@@ -274,7 +293,15 @@ func (db *DB) searchCheckpoint(blockNum uint64, logIdx uint32) (int64, error) {
 	return (i - 1) * searchCheckpointFrequency, nil
 }
 
+func (db *DB) AddDependantLog(log common.Hash, block eth.BlockID, timestamp uint64, logIdx uint32, execMsg ExecutingMessage) error {
```

Comment:
I suspect we should just have one public method for logs with or without an executing message - this just reduced the number of changes required in tests. Will clean up once the parent PR is merged to reduce the number of conflicts to deal with.